config/runtime.exs

```diff
@@ -4,3 +4,12 @@ if config_env() == :prod do
   config :signs_ui, SignsUiWeb.Endpoint,
     secret_key_base: Map.fetch!(System.get_env(), "SECRET_KEY_BASE")
 end
+
+
+if System.get_env("ENVIRONMENT_NAME") == "prod" do
+  config :signs_ui, SignsUiWeb.Endpoint,
+    screenplay_base_url: "https://screenplay.mbta.com/"
+else
+  config :signs_ui, SignsUiWeb.Endpoint,
+    screenplay_base_url: "localhost:4000/ https://screenplay-dev.mbtace.com/"
+end
```

Comment:
Is the fallback URL right? "localhost:4000/ https://screenplay-dev.mbtace.com/"

Also, if you want, you could get the value separately just to avoid some duplicated code:
```suggestion
screenplay_base_url =
  if System.get_env("ENVIRONMENT_NAME") == "prod",
    do: "https://screenplay.mbta.com/",
    else: "localhost:4000/ https://screenplay-dev.mbtace.com/"

config :signs_ui, SignsUiWeb.Endpoint,
    screenplay_base_url: screenplay_base_url
```

Reply:
Yep the fallback URL is right. You can add one or more `frame-ancestors` with just URLs separated by whitespace. The reason I have both localhost and Signs UI Dev is so that both your local instance of Screenplay along with dev will be allowed to embed the Signs UI URLs